deadd-notification-center.cabal

```diff
@@ -76,7 +76,7 @@ executable deadd-notification-center
   main-is:             Main.hs
   ghc-options:         -threaded -rtsopts -with-rtsopts=-N
   build-depends:       base
-                     , linux-notification-center
+                     , deadd-notification-center
```

Comment:
Hi, thank you for the PR. Sadly it does not build. Could you take a look at that?